weaviate/collections/queries/near_vector/generate.py

```diff
@@ -53,6 +55,14 @@ async def near_vector(
         Arguments:
             `near_vector`
                 The vector to search on, REQUIRED. This can be a base64 encoded string of the binary, a path to the file, or a file-like object.
+            `single_prompt`
+                The prompt to use for RaG on each object individually.
+            `grouped_task`
+                The prompt to use for RaG on the entire result set.
```

Comment:
We don't really use RAG in the python client, but generative.

How about something like
`The prompt for the generative query on all results`
(same for the others)